app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+# :nodoc:
+class CommentsController < ApplicationController
```

Comment:
- [ ] Guest can only see sign-in page. We need `authenticate_user!` in comments_controller to permit only authenticated users.